src/templates/landing_pages/ico.html.tt

```diff
@@ -165,7 +165,19 @@
                     </p>
                 </div>
             </div>
-            <hr>
+            <div id="lykke-section" class="content-section center-text fill-bg-color">
+                <div class="container">
+                    <div class="gr-row">
+                        <div class="gr-10 gr-push-1">
+                            <h2>[% l('Trade your tokens on the Lykke Exchange') %]</h2>
+                            <img src="[% request.url_for('images/ico/icons/lykke_logo.svg') %]"/>
+                            <p>[% l('We\'ve reached an agreement to list the tokens from our forthcoming ICO on the [_1]. Lykke is a fintech company that\'s building a global marketplace on the blockchain.', '<a href="https://www.lykke.com/" target="_blank">Lykke Exchange</a>') %]</p>
+                            <a class="button" href="https://www.lykke.com/company/news/binary" target="_blank"><span>[% l('Read the official press release') %]</span></a>
```

Comment:
do we need the noreferrer noopener thing on these links?

Reply:
i think so? to be safe..